e2e/newenemy/newenemy_test.go

```diff
@@ -160,7 +162,8 @@ func TestNoNewEnemy(t *testing.T) {
 		spice.WithMetricsPort(9100),
 		spice.WithDashboardPort(8100),
 		spice.WithDBName(dbName))
-	require.NoError(t, protectedSpiceDB.Start(ctx, tlog, "protected"))
+	require.NoError(t, protectedSpiceDB.Start(ctx, tlog, "protected",
```

Comment:
I think it might be worth having the tests for the non-per-request key remain as well